### PR TITLE
tests: add tests for test utils

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

tests

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
- Adds tests for run util which we use for e2e tests.

**Does this PR introduce a breaking change?**
No

**Other information**
Any other cases which can be tested are appreciated.

### CODE REVIEW

File: test/utils/test-utils.test.js [hunk 1]
```diff
@@ -62,3 +62,38 @@ describe('copyFile', () => {
         });
     });
 });
+
+describe('Runs webpack CLI for a test case correctly', () => {
```
Comment (rishabh3112) [inline test/utils/test-utils.test.js:66]:
```suggestion
describe('run function', () => {
```

File: test/utils/test-utils.test.js [hunk 2]
```diff
@@ -62,3 +62,38 @@ describe('copyFile', () => {
...
+    it('Run function works correctly', () => {
```
Comment (rishabh3112) [inline test/utils/test-utils.test.js:67]:
```suggestion
    it('should work correctly by default', () => {
```

File: test/utils/test-utils.test.js [hunk 3]
```diff
@@ -62,3 +62,38 @@ describe('copyFile', () => {
...
+        // Executes the correct command
+        expect(command).toContain('cli.js');
+        // Should use apply a default output dir
+        expect(command).toContain('--output');
+        expect(command).toContain('bin');
+        expect(stdout).toBeTruthy();
+        expect(stderr).toBeFalsy();
+    });
+
+    it('Run function executes cli with passed commands and params', () => {
```
Comment (rishabh3112) [inline test/utils/test-utils.test.js:78]:
```suggestion
    it('executes cli with passed commands and params', () => {
```

File: test/utils/test-utils.test.js [hunk 4]
```diff
@@ -62,3 +62,38 @@ describe('copyFile', () => {
...
+
+        // Contains info command output
+        expect(stdout).toContain('System:');
+        expect(stdout).toContain('Node');
+        expect(stdout).toContain('npm');
+        expect(stdout).toContain('Yarn');
+        expect(stderr).toBeFalsy();
+    });
+
+    it('Run function uses default output when output param is false', () => {
```
Comment (rishabh3112) [inline test/utils/test-utils.test.js:92]:
```suggestion
    it('uses default output when output param is false', () => {
```

**Comment by anshumanv** [comment]
There you go @rishabh3112 👍 
cc @evilebottnawi

**Comment by anshumanv** [comment]
I initially tested the params in the subprocess but then I realized they work entirely differently in windows hence tests were failing, so made them a bit simpler almost like run tests.

**Pull request merged**